adoc/headers/nditem.h

```diff
@@ -47,6 +47,33 @@ template <int Dimensions = 1> class nd_item {
 
   nd_range<Dimensions> get_nd_range() const;
 
+  // Deprecated in SYCL 2020. 
+  template <typename DataT>
+  device_event async_work_group_copy(local_ptr<DataT> dest,
+                                     global_ptr<const DataT> src,
+                                     size_t numElements) const;
+
+  // Deprecated in SYCL 2020.
+  template <typename DataT>
+  device_event async_work_group_copy(global_ptr<DataT> dest,
+                                     local_ptr<const DataT> src,
+                                     size_t numElements) const;
+
+  // Deprecated in SYCL 2020.
+  template <typename DataT>
+  device_event async_work_group_copy(local_ptr<DataT> dest,
+                                     global_ptr<const DataT> src,
+                                     size_t numElements,
+                                     size_t srcStride) const;
+
+  // Deprecated in SYCL 2020.
+  template <typename DataT>
+  device_event async_work_group_copy(global_ptr<DataT> dest,
+                                     local_ptr<const DataT> src,
+                                     size_t numElements,
+                                     size_t destStride) const;
```

Comment:
Same comment here.  These functions should be added to Table 116 "Member functions for the nd_item class" in section  4.9.1.5.

Reply:
Fixed in https://github.com/KhronosGroup/SYCL-Docs/pull/432/commits/b84963dbb7a353890317fcaa01c8c172a727ae68.